Makefile

```diff
@@ -38,7 +38,7 @@ check-js:
 	(cd ui && yarn testformat)
 
 download-tools:
-	GO111MODULE=off go get -u github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
This changes versions of dependencies defined inside the go.mod, something like this shouldn't be done on the build.

Reply:
Ok, then maybe some dependencies have to be updated since it always fails, even for the CI worker.